quill-spark/src/main/scala/io/getquill/context/spark/SparkDialect.scala

```diff
@@ -33,7 +37,6 @@ class SparkDialect extends SqlIdiom {
         case q: Query =>
           val sql = SqlQuery(q)
           trace("sql")(sql)
-          VerifySqlQuery(sql).map(fail)
```

Comment:
Does it safe to remove this?

Reply:
I'm planning to work on a new check specific for Spark. It has different needs since it allows nested structures. I'll open an issue to follow up on this if you're ok with it.